aktivitetslogg/src/main/kotlin/no/nav/dagpenger/aktivitetslogg/IAktivitetsloggMediator.kt

```diff
@@ -2,28 +2,27 @@ package no.nav.dagpenger.aktivitetslogg
 
 import no.nav.dagpenger.aktivitetslogg.serde.AktivitetsloggJsonBuilder
 
-
-interface AktivitetsloggMessageContext {
-    fun publish(eventNavn: String = "aktivitetslogg", message: Map<String, Any>)
-}
-
-interface AktivitetsloggHendelse : Subaktivitetskontekst {
+interface PersonHendelse : Aktivitetskontekst, IAktivitetslogg {
```

Comment:
Vil alle hendelser ha ident? Blir ikke helt enig med meg selv. Hvis vi skal publisere aktivitetslogg blir den i en kontekst av en person (ident).  Appene må da arve eller delegere PersonHendelse for at det skal funke.

Reply:
@androa Introduserte kommandoer uten ident, men det var slik at en kommando kanskje alltid utføres i en kontekst med ident?

Reply:
Ja, jeg tror det ble konklusjonen. Sjeldent/aldri at vi gjør ting uten at det er på en person.